### PR TITLE
Move approver-policy install/confiugration from usage to projects

### PR DESCRIPTION
**Pull request by JoshVanL**
As [per](https://github.com/cert-manager/website/pull/997#issuecomment-1148447620), this PR moves the approver-policy page into the new `/projects` section. The usages page has been replaced with a small blurb and a link to the projects page.

Also adds a bullet point to the usages `README.md` for approver-policy.

/assign @wallrj

### CODE REVIEW

File: content/docs/projects/README.md
Comment (wallrj) [inline content/docs/projects/README.md:16]: I find it confusing to have the link back to https://github.com/cert-manager/policy-approver here. 
```suggestion
- [approver-policy: ](./approver-policy.md):
  a cert-manager **approver** that will approve or deny CertificateRequests based on CRD defined policies.
```

I suggest having a single link here to the new page on cert-manager.io. And in there we can mention the link to the source code and link to the approver concepts.

File: content/docs/projects/README.md
Comment (wallrj) [inline content/docs/projects/README.md:19]: Similarly: 

```suggestion
- [trust](./trust.md): an 
    operator to distribute trust bundles, like CA certificates, across a
    Kubernetes cluster.
```

File: content/docs/usage/approver-policy.md
Comment (wallrj) [inline content/docs/usage/approver-policy.md:11]: I suggest not linking to the GitHub project here. We want to direct users to the cert-manager.io/projects page if that is to be the new home of the documentation.
```suggestion
approver-policy is a
cert-manager project that enables you to write policy to automatically manage
this approval mechanism.
```

**Review by wallrj — APPROVED** [review]
I like the new projects menu.
Just a couple of suggestions.

/lgtm
/hold in case you want to make the suggested changes.

**Comment by JoshVanL** [comment]
Thanks @wallrj, I have updated. PTAL

File: content/docs/projects/README.md
Comment (wallrj) [inline content/docs/projects/README.md:14]:
```suggestion
- [approver-policy](./approver-policy.md):
  a cert-manager **approver** that will automatically approve or deny
  certificate requests based on defined policy.
```

**Review by wallrj — APPROVED** [review]
Just one tiny nit.

/lgtm
/hold in case you want to also fix the nit

**Review by jahrlin — APPROVED** [review]
Great stuff, very accessible.

/lgtm

**Review by jakexks — COMMENTED** [review]
/lgtm

**Pull request merged**
